src/table/table.scss

```diff
@@ -122,265 +251,316 @@
         }
```

Comment:
Maybe all rules for iui-icon are not needed?
These do not work as we use borderless button (iui-icon is not first level child...)